extension/src/paymentHandler/make-payment.handler.js

```diff
@@ -26,4 +35,16 @@ async function execute (paymentObject) {
   }
 }
 
+async function _fetchMatchingCart (paymentObject) {
```

Comment:
It does not do anything Klarna specific, so I didn't want to put it there.

Reply:
no, It's only klarna specific, fetching cart only needed for klarna make payment. I will resolve this because it's resolved in a different commit: 3676901